float target_cmd  # motor current, in amps